source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -172,9 +172,13 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
     
     private func setupAccessibility() {
         setAccessibilityElement(true)
-        setAccessibilityValue(nil)
+        setAccessibilityValue(config.supportsSchemeV1_3 ? nil : value)
```

Comment:
Why is this nil for 1.3 elements?

Reply:
Because accessibilityLabel takes care of the value as well in 1.3. I have designed it in such a way that in 1.3, everything happens at one place, for easy debugging/changes later